src/scenes/Languages/Detail/LanguageDetail.tsx

```diff
@@ -188,6 +226,13 @@ export const LanguageDetail = () => {
             }
             loading={!language}
           />
+
+          <BooleanProperty
+            label="First Scripture"
+            redacted="You do not have permission to view whether the language has a first Scripture"
+            data={hasFirstScripture}
+            wrap={(node) => <Grid item>{node}</Grid>}
+          />
```

Comment:
I think we need to show more than a boolean here. The options are an engagement, external, or false. It would be useful to show that first scripture is fulfilled by X engagement (name/link) or fulfilled externally, or not fulfilled.

This complexity is why I voted not to have it as a calculated field in the API.